app/controllers/messages_controller.rb

```diff
@@ -146,12 +146,12 @@ def quote
     content = "#{ll(Setting.default_language, :text_user_wrote, user)}\\n> "
     content << text.to_s.strip.gsub(%r{<pre>((.|\s)*?)</pre>}m, '[...]').gsub('"', '\"').gsub(/(\r?\n|\r\n?)/, '\\n> ') + '\\n\\n'
     render(:update) { |page|
-      page << "$('message_subject').value = \"#{subject}\";"
-      page.<< "$('message_content').value = \"#{content}\";"
+      page << "$('reply_subject').value = \"#{subject}\";"
+      page.<< "$('reply_content').value = \"#{content}\";"
       page.show 'reply'
```

Comment:
Also looking totally legit to have that `page.show` between the `page << foo` lines... of course... mixing `page.<<` and `page <<` which is probably the same...

Not that any of this is your fault, but this looks like a big pile-o-WTF...